tools/combine.py

```
@@ -18,7 +19,7 @@
 
 for name in glob.glob("*.txt"):
     with open(name, encoding='utf-8') as f:
-        for l in f.readlines():
+        for l in f:
```

Ruff

Ambiguous variable name: `l`

(E741)